packages/foreman/foreman-bootloaders-redhat/foreman-bootloaders-redhat.spec

```
@@ -11,7 +11,9 @@ BuildArch: noarch
 Source0: foreman-generate-bootloaders
 
 Requires: grub2-tools-extra
```

Further testing shows it wouldn't pull in `grub2-tools` needed for `grub2-mkimge`. I'd suggest
```suggestion
Requires: /usr/bin/grub2-mknetdir
Requires: /usr/bin/grub2-mkimage
```

But then later I found out shim-x64 pulls in grub2-tools.

Still, I like this explicit bit.